tests/unit/drivers/rank/aggregate/test_aggregate_matches_rank_driver.py

```diff
@@ -109,10 +93,10 @@ def test_collect_matches2doc_ranker_driver_mock_ranker():
     driver()
     dm = list(doc.matches)
     assert len(dm) == 2
-    assert dm[0].id == '20' * 8
+    assert dm[0].id == '20'
     assert dm[0].score.value == 3
-    assert dm[1].id == '30' * 8
-    assert dm[1].score.value == 1
+    assert dm[1].id == '30'
+    assert dm[1].score.value == 2
```

Comment:
why does this change?

Reply:
The id of  match5 is set to the same as match4 in the previous code`match5.id = str(4) * 16`